srcpkgs/libircclient/template

```diff
@@ -0,0 +1,54 @@
+# Template file for 'libircclient'
+pkgname=libircclient
+version=1.10
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openssl --enable-ipv6 --enable-shared"
+makedepends="libressl-devel"
+short_desc="IRC client library"
+maintainer="maciozo@maciozo.com <maciozo@maciozo.com>"
+license="LGPL-3.0-or-later"
+homepage="https://www.ulduzsoft.com/linux/libircclient/"
+distfiles="https://netix.dl.sourceforge.net/project/libircclient/libircclient/${version}/libircclient-${version}.tar.gz"
+checksum=bbb26f3af348b252c5204917a7f91cfdf172f1b6afbf4df1e561b03e20503c2d
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)	depends+=" musl-devel";;
+	*)		depends+=" glibc-devel";;
```

Comment:
Don't use tabs except before the first non-blank character of a line.

Reply:
I highly doubt that the libc devel packages are even needed at runtime, these 4 lines can most likely be removed.

Reply:
> I highly doubt that the libc devel packages are even needed at runtime, these 4 lines can most likely be removed.

I'm going by what the `libircclient` docs say. The way I understand it, those are runtime dependancies.